web_app/dist/title.css

```diff
@@ -0,0 +1,47 @@
+html {
+    
+}
+
+body {
+    background: #93ca68;
+    color: white;
+}
+
+.title-page-font {
+    font-family: "Raleway", Helvetica, Arial, sans-serif;
+}
+
+.title-page-spacing {
+    padding: 0;
+    margin: 0;
+}
+
+#title {
+    font-size: 100px;
+}
+
+#tagline {
+    font-size: 30px;
+}
+
+.logo-center {
+    opacity: 0.15;
```

Comment:
Calling this class `logo-center` makes no sense, name it something more applicable

Reply:
"logo" then